README.md

```diff
@@ -1,27 +1,74 @@
+
+If you intend to make open source contributions to the IntelliJ
+Platform, see [Contributing to the IntelliJ Project](http://www.jetbrains.org/display/IJOS/Contribute) for more information.
+
+_**Speed Tip:**_ If the complete repository history isn't
+needed then using a shallow clone (`git clone --depth 1`) will save significant time.
+
+These Git operations can also be done through the [IntelliJ IDEA user interface](https://www.jetbrains.com/help/idea/using-git-integration.html).
+
+IntelliJ IDEA Community Edition requires additional Android modules from
+separate Git repositories. To clone these repositories, run one of the
+`getPlugins` scripts located in the `<IDEA_HOME>` directory. Note: by default these 
+scripts clone their respective *master* branches. You must ensure the branch/tags used for cloning
+these additional repositories matches the branch/tags used for cloning
+the IntelliJ Community Edition source.
+* `getPlugins.sh` for Linux or Mac OS.
+* `getPlugins.bat` for Windows.
+
+## Building IntelliJ Community Edition
+Versions 2017.3 and newer of IntelliJ IDEA Community Edition or IntelliJ
+IDEA Ultimate Edition are required to build and develop for the IntelliJ
+Platform.
+
+### Opening the IntelliJ Source Code for Build
+Using IntelliJ IDEA **File | Open**, select the `<IDEA_HOME>` directory. 
+* If IntelliJ IDEA displays an error about a missing or out of date required plugin (e.g.
+Kotlin), [enable, upgrade, or install that plugin](https://www.jetbrains.com/help/idea/managing-plugins.html) and restart IntelliJ IDEA.
+* If IntelliJ IDEA displays and error about a Gradle configuration not found,
+ [refresh the Gradle projects](https://www.jetbrains.com/help/idea/jetgradle-tool-window.html). 
+
+### IntelliJ Build Configuration
+JDK version 1.8 is required for building and developing for IntelliJ IDEA Community
+Edition.
+1. Using IntelliJ IDEA, [configure](https://www.jetbrains.com/help/idea/sdk.html) a JSDK
+   named "**1.8**", pointing to `<JDK_18_HOME>`.
+   * If not already present, add `<JDK_18_HOME>/lib/tools.jar` [to the Classpath](https://www.jetbrains.com/help/idea/sdk.html#manage_sdks) tab for the **1.8** JSDK.
+2. Also configure a JSDK named "**IDEA jdk**" (case sensitive), pointing to `<JDK_16_HOME>`. If you don’t want to install JDK 1.6 then you may configure **IDEA jdk** to point to `<JDK_18_HOME>`. However, you must be careful to avoid using Java 8 APIs in IntelliJ IDEA Community Edition modules that use **IDEA jdk**. 
+   * If not already present, add the corresponding path for tools.jar to the Classpath for "**IDEA jdk**" JSDK.
+3. [Verify](https://www.jetbrains.com/help/idea/enabling-and-disabling-plugins.html) the _Maven Integration_ plugin is enabled. If not, enable it and [add path variable](https://www.jetbrains.com/help/idea/working-with-projects.html#path-variables) "MAVEN_REPOSITORY" pointing to the `<USER_HOME>/.m2/repository` directory.
```

Comment:
> If not, enable it and

You need to either enable the plugin **or** specify the path variable, see the original text.